hi-ml-cpath/testhisto/testhisto/utils/test_plots_utils.py

```diff
@@ -163,9 +169,10 @@ def test_pr_curve_integration(tmp_path: Path, caplog: pytest.LogCaptureFixture)
         ResultsKey.TRUE_LABEL: [0, 1, 0, 1, 0, 1],
         ResultsKey.PROB: [0.1, 0.8, 0.6, 0.3, 0.5, 0.4]
     }
+    stratify_metadata = ["A", "B", "A", "A", "B", "B"]
 
     # check plot is produced and it has right filename
-    save_pr_curve(results, tmp_path, stage='foo')  # type: ignore
```

Comment:
you might want to keep this as well without stratify_metadata to test the case witout stratification, please feel free to ignore this if it's already tested above

Reply:
good point, I have parametrized `stratify_metadata`